css/style.css

```diff
@@ -14,4 +14,186 @@ a:hover{text-decoration: none;}
 ul li{list-style: none;}
 img{vertical-align: top;}
 h1,h2,h3,h4,h5,h6{font-size:inherit;font-weight: inherit;}
-/*--------------------*/
+/*----------------------------------*/
+
+/*------- technical styles-------*/
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 18px;
+    line-height: 133%;
```

Comment:
It's better to fix this line height by pixels, since 133% may be a decimal number

Reply:
with % for responsibility